sdk/servicefabric/Azure.ResourceManager.ServiceFabric/api/Azure.ResourceManager.ServiceFabric.netstandard2.0.cs

```diff
@@ -425,39 +432,108 @@ public partial class ArmServiceTypeHealthPolicy
         public static bool operator !=(Azure.ResourceManager.ServiceFabric.Models.ArmUpgradeFailureAction left, Azure.ResourceManager.ServiceFabric.Models.ArmUpgradeFailureAction right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class AzureActiveDirectory
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ClusterAddOnFeature : System.IEquatable<Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ClusterAddOnFeature(string value) { throw null; }
+        public static Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature BackupRestoreService { get { throw null; } }
+        public static Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature DnsService { get { throw null; } }
+        public static Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature RepairManager { get { throw null; } }
+        public static Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature ResourceMonitorService { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature left, Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature right) { throw null; }
+        public static implicit operator Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature left, Azure.ResourceManager.ServiceFabric.Models.ClusterAddOnFeature right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ClusterAzureActiveDirectorySetting
```

Comment:
I think we should prepend the RP name on this - could this be `ServiceFabricClusterAadSetting`?
I believe AAD should be a common abbreviation

Reply:
fixed